spicy/src/rt/driver.cc

```diff
@@ -17,7 +21,17 @@ using hilti::rt::fmt;
 
 using namespace spicy::rt;
 
-inline void Driver::_debug(const std::string_view& msg) {
+inline static auto _pretty_print(uint64_t n) {
```

Comment:
Now that this is visible, could we give the function a more descriptive name and also name the argument accordingly, e.g., `auto pretty_print_memory(uint64_t mem)` or similar? I don't think we need to starting underscore as this function is only visible in this TU.